telemetry/definitions/commonDefinitions.json

```diff
@@ -5971,6 +5994,36 @@
                 }
             ]
         },
+        {
+            "name": "languageServer_setup",
+            "description": "LSP setup event",
+            "metadata": [
+                {
+                    "type": "id",
+                    "required": true
```

Comment:
metadata fields are by default true. Only those meant to be optional need the specification of false.